plugins/dell/fu-dell-smi.c

```diff
@@ -169,26 +163,26 @@ fu_dell_query_dock (FuDellSmiObj *smi_obj, DOCK_UNION *buf)
 	result = fu_dell_get_res (smi_obj, cbARG1);
 	if (result != SMI_SUCCESS) {
 		if (result == SMI_INVALID_BUFFER) {
-			g_debug ("Invalid buffer size, needed %" G_GUINT32_FORMAT,
-				 fu_dell_get_res (smi_obj, cbARG2));
+			g_debug (
+			    "Invalid buffer size, needed %" G_GUINT32_FORMAT,
+			    fu_dell_get_res (smi_obj, cbARG2));
 		} else {
-			g_debug ("SMI execution returned error: %d",
-				 result);
+			g_debug ("SMI execution returned error: %d", result);
 		}
 		return FALSE;
 	}
 	return TRUE;
 }
 
-const gchar*
+const gchar *
 fu_dell_get_dock_type (guint8 type)
 {
 	g_autoptr (FuDellSmiObj) smi_obj = NULL;
 	DOCK_UNION buf;
 
 	/* not yet initialized, look it up */
 	if (type == DOCK_TYPE_NONE) {
-		smi_obj = g_malloc0 (sizeof(FuDellSmiObj));
+		smi_obj = g_malloc0 (sizeof (FuDellSmiObj));
```

Comment:
`sizeof()` is really a pre-processor directive, and it does look weird with a space...

Reply:
corrected